CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/delimited/ImportDelimitedPlugin.java

```diff
@@ -412,7 +416,8 @@ private static Map<String, Integer> processVertices(final ImportDefinition defin
                     }                    
                     ++skippedRow;
                 } else {
-                    throw ex;
+                    NotifyDisplayer.display("Unable to complete import due to error with data." + ex.getMessage() +" The file can be imported if you select Skip invalid rows checkbox next to the Import button.", NotifyDescriptor.ERROR_MESSAGE);                    
+                    break;
```

Comment:
Putting a break here causes a change in the functionality.
It will populate the first few results it had collected up to the point where an error occurred, then display only those results in the graph, when there would be many more valid results displayed if the `Skip Invalid Results` checkbox was ticked.
It should _**not**_ return any results if an exception occurs when the `Skip Invalid Results` checkbox is **_not_** ticked.